documentation/src/main/markdown/integrations/azureplugin/azurepluginreleasenotes.md

```diff
@@ -1,10 +1,28 @@
 # Release Notes for Azure DevOps Plugin
 
+## Version 10.0.0
+**Notice**
+
+The [company_name] Software Integrity Group is now [var_company_name]    
+* As part of this activity, sig-repo.synopsys.com and detect.synopsys.com are being deprecated and will be decomissioned in Feb. 2025. Please make use of repo.blackduck.com and detect.blackduck.com respectively.    
+* Refer to the [Black Duck Domain Change FAQ](https://community.blackduck.com/s/article/Black-Duck-Domain-Change-FAQ#SDInstru).
+
+If you are an existing [detect_product_short] user, manually uninstall the [company_name] [detect_product_short] Plugin and install the [detect_product_long] Plugin to receive future updates.    
```

Comment:
@sig-cpotts For existing users thing, highlight more in bold or actions to take based on feedback we got on Jenkins lol 😆